cypress/e2e/cloud/orgs.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Orgs', () => {
     it('forwards the user to the No Orgs Page', () => {
       cy.url().should('contain', 'no-org')
       cy.contains('Sign In').click()
-      cy.url().should('contain', 'dex')
+      cy.location('pathname').should('eq', '/signin')
```

Comment:
firefox doesn't allow the redirect to dex, so lets just check that we make it to the `/signin` page instead